src/MoBi.Presentation/Presenter/SelectReferencePresenterBase.cs

```diff
@@ -167,28 +168,27 @@ private ReferenceDTO timeReference()
       {
          return new ReferenceDTO
          {
-            Path = new TimePath {TimeDimension = _context.DimensionFactory.Dimension(Constants.Dimension.TIME)}
+            Path = new TimePath { TimeDimension = _context.DimensionFactory.Dimension(Constants.Dimension.TIME) }
          };
       }
 
-      public ObjectPath GetSelection()
+      public virtual ObjectPath GetSelection()
       {
          var selection = getSelected<IEntity>();
 
-         return shouldCreateAbsolutePaths ? 
-            _objectPathFactory.CreateAbsoluteObjectPath(selection) : 
-            _objectPathFactory.CreateRelativeObjectPath(_refObject, selection);
+         return shouldCreateAbsolutePath ? _objectPathFactory.CreateAbsoluteObjectPath(selection) : _objectPathFactory.CreateRelativeObjectPath(_refObject, selection);
       }
 
-      private T getSelected<T>() where T : class, IObjectBase
+      protected T getSelected<T>() where T : class, IObjectBase
       {
          var dto = _view.SelectedDTO;
+
          return dto == null ? null : _context.Get<T>(dto.Id);
       }
```

Comment:
`GetSelection` can be non-virtual I think since we changed the DTO behavior. If `getSelected<T>` needs to stay protected then the name has to be capitalized.